build_alpha.sh

```diff
@@ -1,3 +0,0 @@
-#!/bin/bash
-./gradlew clean build
-cp app/build/outputs/apk/app-alpha-matrixorg.apk ./alpha.apk
```

Comment:
right - yup, looks like it's stale from before -console got split out into a separate project. but *please* do separate random stuff like this into seperate PRs in future, as it makes reviewing and merging much easier.